test/test-functions

```diff
@@ -307,7 +307,7 @@ install_dmevent() {
 
 install_systemd() {
     # install compiled files
-    (cd $TEST_BASE_DIR/..; set -x; make DESTDIR=$initdir install)
+    (set -x; DESTDIR=$initdir ninja -C $ABS_BUILD_DIR install)
```

Comment:
I'd rather we fell back on `make install` when `ninja -C install` didn't work for any reason. That simplifies using `git bisect run bash -c '...; make clean setup run; ...'` a bit.